tables/zenodo-upload/README.md

```diff
@@ -1,69 +1,201 @@
-|   S2G     |             |
-|   S2H     |             |
-|   S2I     |             |
-|   S3A     |             |
-|   S3B     |             |
-|   S3C     |             |
-|   S3D     |             |
-|   S3E     |             |
-|   S4A     |             |
-|   S4B     |             |
-|   S5A     |             |
-|   S5B     |             |
-|   S5C     |             |
-|   S6A     |             |
-|   S6B     |             |
-|   S6C     |             |
-|   SD6     |             |
-|   S6E     |             |
-|   S6F     |             |
-|   S7A     |             |
-|   S7B     |             |
-|   S7C     |             |
-|   S7D     |             |
-|   S7E     |             |
-|   S7F     |             |
-|   S7G     |             |
-|   S7H     |             |
-|   S7I     |             |
 
+# Data underlying OpenPBTA Manuscript Figures and Molecular Alterations
 
+This directory contains CSV files that represent data inputted to plots shown in the OpenPBTA manuscript.
+It is intended to facilitate inspection of the underlying data easier for readers and to explicitly capture the samples included in figures (where applicable).
+To **reproduce the figures**, we recommend using the code in the analysis repository: <https://github.com/AlexsLemonade/OpenPBTA-analysis>.
+Please see `figures/` directory documentation in the repository, as well as the documentation for figure generation scripts (`figures/scripts/README.md`).
 
 
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
 
+- [Figure Data](#figure-data)
+  - [Main Figure Data](#main-figure-data)
+    - [Figure 1](#figure-1)
+    - [Figure 2](#figure-2)
+    - [Figure 3](#figure-3)
+    - [Figure 4](#figure-4)
+    - [Figure 5](#figure-5)
+  - [Supplemental Figure Data](#supplemental-figure-data)
+    - [Figure S2](#figure-s2)
+    - [Figure S3](#figure-s3)
+    - [Figure S4](#figure-s4)
+    - [Figure S5](#figure-s5)
+    - [Figure S6](#figure-s6)
+    - [Figure S7](#figure-s7)
+- [Molecular Alterations](#molecular-alterations)
+  - [File Description](#file-description)
+  - [Alteration Notation](#alteration-notation)
+  - [Caveats](#caveats)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Figure Data
+
+Data for the figures below are included as part of this download.
+The table for each figure contains the following:
+
+| Column | Purpose |
+|--------|---------|
+| Figure Panel | Number and letter of the panel(s) |
+| Filename | Name of the file that contains the relevant panel's data |
+| Brief Description | An overview of what the panel includes; please see the manuscript [figure legends](https://alexslemonade.github.io/OpenPBTA-manuscript/#figure-titles-and-legends) and [supplemental figures](https://alexslemonade.github.io/OpenPBTA-manuscript/#supplemental-information-titles-and-legends) for more information |
```

Comment:
We're having all kinds of gh page failures over in `OpenPBTA-manuscript`, so we can't assume that the HTML manuscript is necessarily up to date. Fundamentally this is gh's problem, not ours, so I don't really know how we could change this... So, leave it!

Reply:
It's probably true enough to leave, yes.